Ansible/playbook.yml

```diff
@@ -0,0 +1,100 @@
+- name: Install To Do app on a new web server
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: api_key
+      prompt: Enter the Trello API key
+      private: no
+
+    - name: api_token
+      prompt: Enter the Trello token
+      private: no
```

Comment:
I understand if you wanted to make it easier to enter the variables, but note that your key and token should be considered private.